modules/swagger-codegen/src/main/resources/Java/libraries/rest-assured/JSON.mustache

```diff
@@ -0,0 +1,490 @@
+{{>licenseInfo}}
+
+package {{invokerPackage}};
+
+import com.google.gson.Gson;
```

Comment:
This template should be put in common with okhttp-gson

Reply:
Done